docs/clients/p4rt-ctl.rst

```diff
@@ -2,95 +2,96 @@
       Copyright 2021-2023 Intel Corporation
       SPDX-License-Identifier: Apache-2.0
 
-      Convention for heading levels in documentation:
-
-      =======  Heading 0 (reserved for the title in a document)
-      -------  Heading 1
-      ~~~~~~~  Heading 2
-      +++++++  Heading 3
-      '''''''  Heading 4
-
-      Avoid deeper levels because they do not render well.
-
 =====================
 p4rt-ctl Client Guide
 =====================
 
-This document describes how to use executables generated by networking-recipe
-for supporting P4 programs.
-
-- For code clone and build instructions refer to ``README.build.md``.
+This document describes how to use P4 Control Plane executables to support
```

Comment:
P4 Control plane executables indicate a broad spectrum. This document only talks about P4runtime Client  (p4rt-ctl) and tdi_pipeline builder.

Reply:
I've been changing _networking-recipe_ to _P4 Control Plane_ since I learned that this package is also a component of the K8S recipe.

It's not incorrect to use the collective term; it's a form of abstraction. One might say "I used my tools to fix the toaster" instead of "I used a small flat-head screwdriver and a pair of needle-nose pliers to fix the toaster."

You have a point, though. It's worth revisiting the wording. I'll save that for a later revision. This pass is mostly about moving toward a common appearance (and cleaning up the RST markup, and fixing broken links).